### ISSUE TITLE
detect when a method doesn't return valid BedTool-supported output

### ISSUE THREAD

**Issue by daler** [daler/pybedtools#113]
`BedTool.genome_coverage` sometimes returns bedGraph-like output, sometimes not.  In fact, by default, it doesn't return a valid BED-ish format file.

See #110, #111, #112.

_note to future self: this prob needs yet another kwarg to `bedtool._wraps`, and you will need to go through BEDTools programs and args to work out which don't create valid output files._

**Comment by laserson** [comment]
Perhaps at least add a note in the docs?  Less confusing that way...

**Comment by daler** [comment]
Yep, good idea. Did you hit this with `genome_coverage` specifically or with another method?

**Comment by laserson** [comment]
Just `genome_coverage` for me so far

**Comment by daler** [comment]
#213 adds some documentation. I will try implementing something like https://github.com/daler/pybedtools/issues/110#issuecomment-206426464, xref #167

**Comment by ajsams** [comment]
I believe that I'm coming across a version of this issue trying to call `coverage` with `hist=True`.  Have tested my inputs in `bedtools` (2.29.2) and they work, but calling with `pybedtools` (0.8.1) leads to `OverflowError: value too large to convert to CHRPOS`
